src/amchart/Candle.js

```diff
@@ -13,13 +13,12 @@
         
         this._gType = "candlestick";
         
-        this._openField;
-        this._closeField;
-        this._categoryField;
-        this._lowField;
-        this._highField;
-    };
-    
+        this._openField = undefined;
+        this._closeField = undefined;
+        this._categoryField = undefined;
+        this._lowField = undefined;
+        this._highField = undefined;
```

Comment:
actually we want to init those are empty arrays as those are private vars (not publish params)